PayPal/src/main/java/com/braintreepayments/api/PayPalVaultRequest.java

```diff
@@ -30,6 +32,20 @@ public boolean getShouldOfferCredit() {
         return shouldOfferCredit;
     }
 
+    /**
+     * Optional: User email to initiate a quicker authentication flow in cases where the user has a
+     * PayPal Account with the same email.
+     *
+     * @param userAuthenticationEmail - email address of the payer
+     */
+    public void setUserAuthenticationEmail(String userAuthenticationEmail) {
```

Comment:
```suggestion
    public void setUserAuthenticationEmail(@Nullable String userAuthenticationEmail) {
```

Reply:
I brought this up on the initial PR and @tdchow found that [it wasn't necessary](https://github.com/braintree/braintree_android/pull/918#discussion_r1486903583) - thoughts?

Reply:
Oh ok maybe it's changed we should check the type signature of this property on the Kotlin side to be sure. Without annotations it used to default to `String!!` rather than `String?`.

Reply:
I just did some testing. Looks like there's a runtime exception thrown when trying to set `userAuthenticationEmail` to null. I thought I'd be good to ensure a merchant passes in a non null value here, and I thought we'd get a compile time check here. 

I'm good to add the nullable annotations to prevent this.